packages/publicui/package.json

```diff
@@ -58,8 +58,10 @@
     "@plone/registry": "workspace:*",
     "@plone/slots": "workspace:*",
     "@plone/theming": "workspace:*",
+    "react-aria-components": "catalog:",
     "react-i18next": "catalog:",
     "react-router": "catalog:",
+    "react-router-dom": "catalog:",
```

Comment:
Btw, https://www.npmjs.com/package/react-router-dom is not needed anymore, they integrated it into `react-router`.